### ISSUE TITLE
Sprint 1.27.0 3/2 - Jul 5 to Jul 16 (Funnels #2)

### ISSUE THREAD

**Issue by mariusandra** [PostHog/posthog#4968]
Last sprint: https://github.com/PostHog/posthog/issues/4765

### Support heros

Paolo + Sam

### Core Experience

- Sam - Funnel visualisation and filtering/taxonomy improvements (breakdowns, horizontal funnels, time visualisations, etc) https://github.com/PostHog/posthog/issues/4535
- Li - Funnels modals (people, dropoffs, filters etc) https://github.com/PostHog/posthog/issues/4819 
- Paolo - support hero, feedback / working sessions on funnels, more calls and usability tests to get feedback. Also (lower priority): https://github.com/PostHog/posthog/issues/4821 ?
- Alex - off 1 week, will continue on https://github.com/PostHog/posthog/issues/4855 or other priority
- Marius - off 1 week, will pick up any priority when back
- Chris - Funnel visualisation, taxonomy and filtering

### Core Analytics
**Eric:**
- Finish breakdown (#4949 
- Breakdown by cohort (#5053)
- Breakdown people (#5043 thanks neil)
- Assist integration of all the new features
- Pinpointing what aspects would improve these query times dramatically (get TODOs in priority order to reduce unnecessary work/changes)

**Michael:**
- Funnel trends people (#4918)
- Funnels trends breakdown (#4989)
- Better histogram binning (#4990)
- Histogram people (#4991)
- Funnel attempt-based (as opposed to person-based) conversion analysis (#4972)

**Neil:**
- Event exclusion
- Making funnel ordering and funnel visualizations work together. Make base query handle funnel ordering (#4988)
- CSV raw data export

### Extensibility

Priorities
- Snowflake 
- Sentry Issues (particularly session recording related ones)
- `posthog-js` session recording issues
- https://github.com/PostHog/plugin-server/issues/484
- Plugins frontend

Side quests
- Feature flags for mobile libs with freelance devs
- https://github.com/PostHog/plugin-server/issues/477
- Increment to `posthog-js`
- Make sure all endpoints with Personal API Key auth support specifying a team (related to https://github.com/PostHog/posthog/issues/4849)

### Deployment & Infrastructure

![image](https://user-images.githubusercontent.com/1727427/124298581-b4b27300-db5c-11eb-8f29-7e7526341d14.png)

**Comment by mariusandra** [comment]
### Core Experience planning (WIP)

Plenty of tasks got unlocked this week, which we'll start integrating. Two people are off for one week.

- Sam - Funnel visualisation and filtering/taxonomy improvements (breakdowns, horizontal funnels, time visualisations, etc) https://github.com/PostHog/posthog/issues/4535
- Li - Funnels modals (people, dropoffs, filters etc) https://github.com/PostHog/posthog/issues/4819 
- Paolo - more calls and usability tests to get feedback on funnels, also: https://github.com/PostHog/posthog/issues/4821 ?
- Alex - off 1 week, will continue on https://github.com/PostHog/posthog/issues/4855 or other priority
- Marius - off 1 week, will pick up any priority when back

**Comment by EDsCODE** [comment]
**Eric:**
- Finish breakdown
- Breakdown by cohort
- Breakdown people
- Assist integration of all the new features

**Michael:**
- Funnel trends people
- Funnels trends breakdown
- Histogram + autobinning
- Histogram people
- Surface how many times people are re-trying the funnel as well as the time taken.  Just a time thing.

**Neil:**

- Event exclusion
- Making funnel ordering and funnel visualizations work together. Make base query handle funnel ordering
- CSV raw data export

**Performance improvements:**

- Pinpointing what aspects would improve these query times dramatically (get TODOs in priority order to reduce unnecessary work/changes)

**Comment by yakkomajuri** [comment]
# Extensibility

<details>
<summary>
<b>Pre-Sprint Planning</b>
</summary>

## TL;DR

### Priorities

- Snowflake 
- Sentry Issues (particularly session recording related ones)
- `posthog-js` session recording issues
- https://github.com/PostHog/plugin-server/issues/484
- Plugins frontend

### Side quests

- Feature flags for mobile libs with freelance devs
- https://github.com/PostHog/plugin-server/issues/477
- Increment to `posthog-js`
- Make sure all endpoints with Personal API Key auth support specifying a team (related to https://github.com/PostHog/posthog/issues/4849)

-----

## Full list

### Priorities

- Snowflake (if @Twixes doesn't pick it up)
- Sentry Issues (particularly session recording related ones)
- `posthog-js` session recording issues
- https://github.com/PostHog/plugin-server/issues/484
- Make sure all endpoints with Personal API Key auth support specifying a team (related to https://github.com/PostHog/posthog/issues/4849)
- Plugins frontend - potential things to work on:
    - App: https://github.com/PostHog/plugin-server/issues/413
    - App: Remove onEvent plugins from ordering
    - App: https://github.com/PostHog/plugin-server/issues/412
    - App: https://github.com/PostHog/plugin-server/issues/411
    - Website: Fix https://posthog.com/plugins dark mode
    - Website: Add individual plugin pages with build time fetching (instead of modal) https://posthog.com/plugins

### Side quests

- Feature flags for mobile libs with freelance devs
- https://github.com/PostHog/plugin-server/issues/477
- Increment to `posthog-js`

</details>

## Post-Sprint Highlights

According to priorities:

### Data Warehouses

- Added support for Snowflake export via S3 and GCS stages
- New GCS export plugin
- Updates to Postgres, RudderStack, and BQ plugins
- Foundation for import plugins 

### Libraries

- `posthog-js` updates: increment, session recording reload, cleanup
- `posthog-php` feature flags and misc improvements
- Tiny updates to `posthog-node` and `posthog-ruby`

### Plugin Server Maintenance

- Shipped fix for https://github.com/PostHog/plugin-server/issues/487
- https://github.com/PostHog/plugin-server/issues/484
- Updated and shipped https://github.com/PostHog/plugin-server/pull/486
- A few Sentry issue fixes

### New developments

- Plugins UI revamp + plugins out of Beta
- New cache methods
- New config options 

### Other 

- Allow specifying a project on API endpoints

### [Full List](https://github.com/PostHog/plugin-server/issues/492)

**Comment by fuziontech** [comment]
## Infra and Deploys

### Top Priorities

1. Netdata performance debugging and resolving
1. Unblock entry into GCP Marketplace
1. Land DigitalOcean Kubernetes one-click deploy product
1. Docs for standing up open source helm chart (self-service install)
1. Docs for standing up OS helm chart on AWS (a lot of crossover with previous one but written specifically for AWS)
1. Support customers using OS helm chart
1. Scope out work for enabling speed-ups by `core-analytics`

![image](https://user-images.githubusercontent.com/391319/124296014-5d85b180-db0e-11eb-9b3d-2fa08fae7d63.png)
